PurchasesTests/Purchasing/RCIntroEligibilityTests.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  RCIntroEligibilityTests.swift
+//  PurchasesTests
+//
+//  Created by Andrés Boedo on 7/1/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import XCTest
+import Nimble
+@testable import Purchases
+
+class RCIntroEligibilityTests: XCTestCase {
```

Comment:
tests here serve as an integration test between swift and obj-c: we're matching `IntroElegibilityStatus` with `RCIntroElegibilityStatus`